qa/rpc-tests/wallet_listunspent.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+# Copyright (c) 2016-2022 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    get_coinbase_address,
+    nuparams,
+    start_nodes,
+    wait_and_assert_operationid_status,
+    NU5_BRANCH_ID
+)
+
+from decimal import Decimal
+
+def unspent_total(unspent):
+    return sum((item['amount'] for item in unspent))
+
+class WalletListUnspent(BitcoinTestFramework):
+    def setup_nodes(self):
+        return start_nodes(4, self.options.tmpdir, [[
+            nuparams(NU5_BRANCH_ID, 201),
+        ]] * 4)
+
+    def matured_at_height(self, height):
+        return unspent_total(self.nodes[0].listunspent(1, 999999, [], height))
+
+    def run_test(self):
+        assert_equal(self.nodes[0].getbalance(), 250)
+        assert_equal(self.nodes[1].getbalance(), 250)
```

Comment:
```suggestion
```suggestion
    def run_test(self):
        def expected_matured_at_height(height):
            return (height-200)*10 + 250

        assert_equal(self.nodes[0].getbalance(), expected_matured_at_height(200))
        assert_equal(self.nodes[1].getbalance(), expected_matured_at_height(200))
```